jenkins_build.sh

```diff
@@ -10,5 +10,6 @@ easy_install nose
 easy_install mock
 easy_install requests
 easy_install coverage
+easy_install pytest
```

Comment:
It is, I wasn't able to run the tests without this line, which is why I added it.